### PR TITLE
README.md updated

### PR DESCRIPTION
**Pull request by hannahy00**
Added a description of project and contributions to the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+The main idea of this project is **inclusive unconferencing**. An unconference is a participant-driven event where anyone can speak, but that's not easy for everyone. Speak-able allows less confident participants to safely gauge interest in a potential topic before scheduling a talk. This increases confidence that people want to hear them and allows them to participate more fully in the event. Everyone benefits when introverts, new speakers and anyone usure they fit in are encouraged to present their ideas.
+
+## How it Works ## 
+Speak-able is a website that allows someone create a room for an unconference or join an existing one using a unique code. It also allows people to submit a topic for other unconference participants to vote on. This allows the potential speaker to gauge interest in their topic and feel confident that people want to hear them speak. 
+
+## Other information ##
+The design was created in Figma, then re-created with HTML/CSS, and Javascript. The back-end is node, express, and Javascript. See the [DevPost posting](https://devpost.com/software/speak-able-inclusive-unconferencing) for more details and a demo about this project. 
+
+**Contributions:**
```
Comment (Green-Avocado) [inline README.md:13]:
```suggestion
### Contributions:
```

just a minor formatting detail

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+
+## Description ##
+The main idea of this project is **inclusive unconferencing**. An unconference is a participant-driven event where anyone can speak, but that's not easy for everyone. Speak-able allows less confident participants to safely gauge interest in a potential topic before scheduling a talk. This increases confidence that people want to hear them and allows them to participate more fully in the event. Everyone benefits when introverts, new speakers and anyone usure they fit in are encouraged to present their ideas.
+
+## How it Works ## 
+Speak-able is a website that allows someone create a room for an unconference or join an existing one using a unique code. It also allows people to submit a topic for other unconference participants to vote on. This allows the potential speaker to gauge interest in their topic and feel confident that people want to hear them speak. 
+
+## Other information ##
+The design was created in Figma, then re-created with HTML/CSS, and Javascript. The back-end is node, express, and Javascript. See the [DevPost posting](https://devpost.com/software/speak-able-inclusive-unconferencing) for more details and a demo about this project.
```
Comment (Green-Avocado) [inline README.md:11]:
```suggestion
The main idea of this project is **inclusive unconferencing**.
An unconference is a participant-driven event where anyone can speak, but that's not easy for everyone.
Speak-able allows less confident participants to safely gauge interest in a potential topic before scheduling a talk.
This increases confidence that people want to hear them and allows them to participate more fully in the event.
Everyone benefits when introverts, new speakers and anyone usure they fit in are encouraged to present their ideas.

## How it Works ## 
Speak-able is a website that allows someone create a room for an unconference or join an existing one using a unique code.
It also allows people to submit a topic for other unconference participants to vote on.
This allows the potential speaker to gauge interest in their topic and feel confident that people want to hear them speak. 

## Other information ##
The design was created in Figma, then re-created with HTML/CSS, and Javascript.
The back-end is node, express, and Javascript. See the [DevPost posting](https://devpost.com/software/speak-able-inclusive-unconferencing) for more details and a demo about this project. 
```

Separating sentences will make future changes easier to compare

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+
+**Contributions:** 
+
+Brian Ho: Overall front-end development, media design, graphics, video recording for the demo and active idea input.
+
+Hannah Yang: Overall front-end development, graphics and design. Was also active in idea making.
+
+Jason Ngo: Created back-end and API to create and manage rooms. Wrote JavaScript to allow users to interact with the API.
+
+Jason Hall: Overall concept, media design, pitch, graphics.
```
Comment (Green-Avocado) [inline README.md:21]:
```suggestion
* Brian Ho: Overall front-end development, media design, graphics, video recording for the demo and active idea input.

* Hannah Yang: Overall front-end development, graphics and design. Was also active in idea making.

* Jason Ngo: Created back-end and API to create and manage rooms. Wrote JavaScript to allow users to interact with the API.

* Jason Hall: Overall concept, media design, pitch, graphics.
```

use bullet points

**Review by Green-Avocado — CHANGES_REQUESTED** [review]
Looks good Hannah, just wanted to suggest some minor changes for formatting.

**Comment by hannahy00** [comment]
Thanks! committed them :)

**Review by Green-Avocado — APPROVED** [review]
Looks good to me! :+1:

**Pull request merged**